package.json

```diff
@@ -136,7 +136,7 @@
     "ngx-base": "2.3.1",
     "ngx-bootstrap": "1.8.0",
     "ngx-fabric8-wit": "6.20.1",
-    "ngx-forge": "0.0.39-development",
+    "ngx-forge": "0.0.48-development",
```

Comment:
Would you please update this to ngx-forge@0.0.52-development?

Can you also please merge my latest changes with your branch? I have a few fixes that are required for these changes to work properly in OSIO.

https://github.com/mindreeper2420/fabric8-ui/pull/6